src/styles/blocks/header.scss

```diff
@@ -0,0 +1,74 @@
+.header {
+  justify-content: space-between;
+  display: flex;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  width: 100%;
+  background-color: $main-background-color;
+
+  &__logo {
+    align-self: center;
+    margin-left: 50px;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  display: flex;
+  margin-right: 50px;
```

Comment:
Don't use margins on BEM block styles

Reply:
> Don't use margins on BEM block styles

Not fixed, BEM block couldn't have any margins or position